### PR TITLE
fix: update query param defs for categories without them

### PR DESCRIPTION
**Pull request by mcstover**
Did a pass in dev with a logged in and logged out user to find loan channels/categories that didn't have a translated url to lend/filter or legacy lend. This fixes a few of those and adds another exclude query param.

### CODE REVIEW

**Comment by mcstover** [comment]
Sorry about the formatting change :(

File: src/plugins/loan-channel-query-map.js [hunk 1]
```diff
@@ -265,7 +315,47 @@ export default {
...
+					queryParams: 'status=fundRaising&sector=7',
+					flssLoanSearch: {
+						sectorId: [7]
+					},
+				},
+				{
+					id: 79,
+					url: 'group-loans',
+					queryParams: 'status=fundRaising&isGroup=1&distributionModel=both',
+					// flssLoanSearch: {},
```
Comment (dyersituations) [inline src/plugins/loan-channel-query-map.js:358]: Remove?

File: src/plugins/loan-channel-query-map.js [hunk 2]
```diff
@@ -375,7 +507,16 @@ export default {
...
+					flssLoanSearch: {
+						themeId: [36],
+						distributionModel: 'FIELDPARTNER'
+					},
+				},
+				{
+					id: 108,
+					url: 'recommended-by-lenders',
+					queryParams: 'status=fundRaising&riskRating=1,5&lenderTerm=0,10&partner=33,465,265,269,116,271,409,438,34,282,444,310,159,210,268,104,445,32,155,108,158,127,43,106,2,3,4,5,128,36,40,109,355,46,105,37,270,287,356,421,346,107,281,286,153,607,41,47,605,111,110,354,280,288,424,447,347,6,472,112,186,437,162,360,164,184,278,313,267,45,365,478,433,96,290,42,1,8,49,353,7,10,219,53,224,188,427,350,227,9,54,50,55,56,370,11,152,52,212,307,136,51,57,364,291,189,369,481,61,168,220,293,113,165,63,299,384,225,533,131,132,476,480,380,135,304,528,172,170,397,521,66,130,295,306,468,67,144,385,141,228,388,393,137,279,396,559,171,344,377,454,289,245,302,341,455,382,403,477,316,395,441,450,300,71,138,142,392,296,70,13,574,262,567,416,229,214,60,64,247,230,248,244,484,312,321,401,143,198,391,238,241,242,305,213,323,72,200,73,16,82,237,485,249,84,324,487,572,415,254,258,462,407,202,327,332,79,204&defaultRate=,0.01&distributionModel=field_partner',
+					// flssLoanSearch: {},
```
Comment (dyersituations) [inline src/plugins/loan-channel-query-map.js:519]: Remove?

**Review by dyersituations — APPROVED** [review]

File: src/plugins/loan-channel-query-map.js (hunk 1 above)
Comment (mcstover) [reply src/plugins/loan-channel-query-map.js:358]: I just left as a placeholder and figured we'd add it once we include that param.

**Pull request merged**